core/src/main/java/org/javarosa/core/model/instance/FormInstance.java

```diff
@@ -270,6 +270,14 @@ public DataInstance initialize(InstanceInitializationFactory initializer, String
         return new String[]{META_XMLNS, META_ID};
     }
 
+    public Hashtable getMetaData() {
```

Comment:
Can you add a `@SuppressWarnings("unused")` annotation so that we know this is used by something outside of odk?